### PR TITLE
ML projects added

### PR DESCRIPTION
**Pull request by chandan-urs**
### Issue Reference
This PR addresses the changes to 

* Change 1
* Change 2

### Summarize
Mention the keypoints here

### CODE REVIEW

File: examples/linear_regression_fortune500/README.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+## Linear Regression Fortune 500 - PySpark ML Project
+
+#### Installation:
+```python
+pip install pysparkcli
```
Comment (jinojossy93) [inline examples/linear_regression_fortune500/README.md]: This command is wrong
Update as following:

`pip install pyspark-cli`

File: examples/linear_regression_income_prediction/README.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
+## Linear Regression Income Prediction - PySpark ML Project
+
+#### Installation:
+```python
+pip install pysparkcli
```
Comment (jinojossy93) [inline examples/linear_regression_income_prediction/README.md]: Same as above

File: examples/parquet_tweets/README.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
+### Parquet Tweets - Pyspark streaming project using parquet files.
+
+#### Installation:
+```python
+pip install pysparkcli
```
Comment (jinojossy93) [inline examples/parquet_tweets/README.md]: Correct this one too. 

We merged these commits already, right?

Sync with master and correct the install command

File: examples/linear_regression_fortune500/README.md (hunk 1 above)
Comment (chandan-urs) [reply examples/linear_regression_fortune500/README.md]: Correcting it now.

File: examples/parquet_tweets/README.md (hunk 3 above)
Comment (chandan-urs) [reply examples/parquet_tweets/README.md]: Sure

File: examples/linear_regression_income_prediction/README.md [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+
+#### Note:
+The input data for the project must be placed in src/inputData folder.
+
+### References:
+https://towardsdatascience.com/machine-learning-at-scale-with-apache-spark-mllib-python-example-b32a9c74c610
+
+## Note to Jino
+
+The last commented statement in app.py is throwing StringIndexer encountered NULL value. Can we ignore this statment?
```
Comment (chandan-urs) [inline examples/linear_regression_income_prediction/README.md]: Please check this

File: examples/linear_regression_fortune500/src/app.py [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+    company_df = company_df.withColumn('Number of Employees', company_df['Number of Employees'].cast("int"))
+    company_df.show(10)
+
+    company_df.cache()
+    company_df.printSchema()
+
+    company_df.describe().toPandas().transpose()
+
+
+    company_df.cache()
```
Comment (jinojossy93) [inline examples/linear_regression_fortune500/src/app.py]: remove this line. This was already cached in line 20

Comment (jinojossy93) [reply examples/linear_regression_fortune500/src/app.py]: Also remove second  printSchema

File: examples/linear_regression_income_prediction/README.md (hunk 4 above)
Comment (jinojossy93) [reply examples/linear_regression_income_prediction/README.md]: Will check this later. Lets hold this PR for now.

Comment (chandan-urs) [reply examples/linear_regression_income_prediction/README.md]: Sure. FYI I have made the changes mentioned in your previous comments.

**Comment by jinojossy93** [comment]
@chandan-urs,

Please resolve the conflicts

**Pull request merged**